articles/zh/procedural-macros-in-rust.md

```diff
@@ -1167,19 +1175,19 @@ version = "0.1.0"
 edition = "2021"
 resolver = "2"
 
-# 更多键及其定义请参见 https://doc.rust-lang.org/cargo/reference/manifest.html
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
```suggestion
# 更多键及其定义请参见  https://doc.rust-lang.org/cargo/reference/manifest.html
```

Reply:
this line is generated by `cargo new` command, not by the author

Reply:
Thank you for explaining. So it shouldn't be translated.